mevshare/sim_result_backend.go

```diff
@@ -45,6 +47,29 @@ func NewSimulationResultBackend(log *zap.Logger, hint HintBackend, builders Buil
 	}
 }
 
+func izZeroPriorityFeeTX(bundle *SendMevBundleArgs) bool {
+	if len(bundle.Body) != 1 {
```

Comment:
should we check if input bundle is nil?

Reply:
overall in codebase we rely on it being non-nil by design